schema/me/conversation/index.js

```diff
@@ -167,12 +167,36 @@ export const ConversationFields = {
   },
   last_message: {
     type: new GraphQLNonNull(GraphQLString),
+    description: "This is a snippet of text from the last message.",
     resolve: conversation => {
       return get(conversation, "_embedded.last_message.snippet")
     },
   },
   last_message_at: date,
 
+  // This is our desired API for an unread indicator, for a given email:
+  // If the given email address is a recipient of the last message, return true.
+  is_last_message_to: {
+    type: GraphQLBoolean,
+    args: {
+      email_address: {
+        type: new GraphQLNonNull(GraphQLString),
+      },
+    },
+    resolve: (conversation, { email_address }) => false, // eslint-disable-line no-unused-vars
+  },
+  // If the given email is a recipient of the last message, return their timestamped
+  // 'read' event, otherwise null.
+  last_message_read_at: {
+    type: GraphQLString,
+    args: {
+      email_address: {
+        type: new GraphQLNonNull(GraphQLString),
+      },
+    },
+    resolve: (conversation, { email_address }) => null, // eslint-disable-line no-unused-vars
+  },
```

Comment:
<img width="398" alt="screen shot 2017-08-10 at 3 27 49 pm" src="https://user-images.githubusercontent.com/1457859/29188236-7c9c437a-7de0-11e7-9cd2-f9e56ab6b182.png">

A test showing how it's called would definnitely help, you're right. Here's a screenshot in the meantime ^